.github/dependabot.yml

```diff
@@ -1,38 +1,51 @@
 version: 2
 updates:
   - package-ecosystem: "npm"
+    allow:
+      - dependency-type: "all"
     directory: "/"
     schedule:
       interval: "weekly"
     groups:
-      nodejs-dependencies:
+      npm-minor:
         patterns:
-        - "*"
+          - "*"
+        update-types:
+          - "minor"
+          - "patch"
 
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "weekly"
 
-  - package-ecosystem: "rust"
+  - package-ecosystem: "cargo"
+    allow:
+      - dependency-type: "all"
     directory: "/"
     schedule:
       interval: "weekly"
     groups:
-      winit-wgpu-dependencies:
+      winit-wgpu-egui:
         patterns:
         - "winit"
         - "wgpu"
         - "naga"
         - "naga_oil"
-        - "egui-winit"
-        - "egui-wgpu"
+        - "egui*"
         - "raw-window-handle"
       wasm-bindgen-dependencies:
```

Comment:
I guess the `-dependencies` part is also really redundant here, but oh well...
Can be fixed next time someone touches it.

Reply:
Turns out, that someone is me... :clown_face: 
https://github.com/ruffle-rs/ruffle/pull/15873